static-html/service.html

```diff
@@ -5,18 +5,18 @@
         <meta http-equiv="X-UA-Compatible" content="IE=edge" />
         <meta name="viewport" content="width=device-width, initial-scale=1" />
         <title>Service Unavailable</title>
-        <link rel="stylesheet" href="https://cdn.ons.gov.uk/sdc/design-system/44.0.2/css/main.css" />
-        <link rel="stylesheet" media="print" href="https://cdn.ons.gov.uk/sdc/design-system/44.0.2/css/print.css" />
+        <link rel="stylesheet" href="https://cdn.ons.gov.uk/sdc/design-system/58.0.2/css/main.css" />
+        <link rel="stylesheet" media="print" href="https://cdn.ons.gov.uk/sdc/design-system/58.0.2/css/print.css" />
         <meta name="theme-color" content="#206095" />
 
         <!-- Favicons -->
-        <meta name="msapplication-config" content="https://cdn.ons.gov.uk/sdc/design-system/44.0.2/favicons/main/en/browserconfig.json" />
-        <link rel="icon" type="image/x-icon" href="https://cdn.ons.gov.uk/sdc/design-system/44.0.2/favicons/main/en/favicon.ico" />
-        <link rel="icon" type="image/png" href="https://cdn.ons.gov.uk/sdc/design-system/44.0.2/favicons/main/en/favicon-32x32.png" sizes="32x32" />
-        <link rel="icon" type="image/png" href="https://cdn.ons.gov.uk/sdc/design-system/44.0.2/favicons/main/en/favicon-16x16.png" sizes="16x16" />
-        <link rel="mask-icon" href="https://cdn.ons.gov.uk/sdc/design-system/44.0.2/favicons/main/en/safari-pinned-tab.svg" color="#000000" />
-        <link rel="apple-touch-icon" type="image/png" href="https://cdn.ons.gov.uk/sdc/design-system/44.0.2/favicons/main/en/apple-touch-icon.png" sizes="180x180" />
-        <link rel="manifest" href="https://cdn.ons.gov.uk/sdc/design-system/44.0.2/favicons/main/en/manifest.json" />
+        <meta name="msapplication-config" content="https://cdn.ons.gov.uk/sdc/design-system/58.0.2/favicons/main/en/browserconfig.json" />
+        <link rel="icon" type="image/x-icon" href="https://cdn.ons.gov.uk/sdc/design-system/58.0.2favicons/main/en/favicon.ico" />
+        <link rel="icon" type="image/png" href="https://cdn.ons.gov.uk/sdc/design-system/58.0.2/favicons/main/en/favicon-32x32.png" sizes="32x32" />
+        <link rel="icon" type="image/png" href="https://cdn.ons.gov.uk/sdc/design-system/58.0.2/favicons/main/en/favicon-16x16.png" sizes="16x16" />
+        <link rel="mask-icon" href="https://cdn.ons.gov.uk/sdc/design-system/58.0.2/favicons/main/en/safari-pinned-tab.svg" color="#000000" />
+        <link rel="apple-touch-icon" type="image/png" href="https://cdn.ons.gov.uk/sdc/design-system/58.0.2/favicons/main/en/apple-touch-icon.png" sizes="180x180" />
+        <link rel="manifest" href="https://cdn.ons.gov.uk/sdc/design-system/58.0.2/favicons/main/en/manifest.json" />
```

Comment:
```suggestion
        <meta name="msapplication-config" content="https://cdn.ons.gov.uk/sdc/design-system/58.0.2/favicons/browserconfig.json" />
        <link rel="icon" type="image/x-icon" href="https://cdn.ons.gov.uk/sdc/design-system/58.0.2/favicons/favicon.ico" />
        <link rel="icon" type="image/png" href="https://cdn.ons.gov.uk/sdc/design-system/58.0.2/favicons/favicon-32x32.png" sizes="32x32" />
        <link rel="icon" type="image/png" href="https://cdn.ons.gov.uk/sdc/design-system/58.0.2/favicons/favicon-16x16.png" sizes="16x16" />
        <link rel="mask-icon" href="https://cdn.ons.gov.uk/sdc/design-system/58.0.2/favicons/safari-pinned-tab.svg" color="#000000" />
        <link rel="apple-touch-icon" type="image/png" href="https://cdn.ons.gov.uk/sdc/design-system/58.0.2/favicons/apple-touch-icon.png" sizes="180x180" />
        <link rel="manifest" href="https://cdn.ons.gov.uk/sdc/design-system/58.0.2/favicons/manifest.json" />
```

These links need updating

Reply:
Tested and re run. All works, thanks !